artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/components/browse.js

```diff
@@ -528,6 +528,8 @@ var Artemis;
 
         var routingTypes = ["multicast", "anycast"];
 
+        var mqttQosTypes = ["at most once", "at least once", "exactly once"];
```

Comment:
I think the numeral should be retained here along with the description, e.g.:
```java
var mqttQosTypes = ["0 - at most once", "1 - at least once", "2 - exactly once"];
```
This is because lots of MQTT documentation, tutorials, and even the specification refer to QoS values directly by their numeral without any description so lots of folks more readily recognize the numeral. Also, lots of APIs use an integer for the QoS value so many developers will be accustomed to this representation as well.

Reply:
@jbertram the formatting function does that always, see the screen print in https://issues.apache.org/jira/browse/ARTEMIS-4120